### DISCUSSION TITLE
It would be great to pull the latest monaco binary in a docker image. Multiple usecases to tackle.

### DISCUSSION THREAD

**Discussion by sergiohinojosa** [Dynatrace/dynatrace-configuration-as-code#336] [Ideas]
Hi guys/girls,

@henrikrexed wants to build a monaco container for scraping prometheus data. I think is a great idea and also a lot of other usecases where monaco users can run the latest monaco without having to install it solving the problem of having outdated monacos.

Adding a build and push step to the integration test could be a nice way to handle this. Then adding some documentation how to run monaco from the container.

What do you think?

Best regards
Sergio

**Comment by kristofre** [comment]
Hi @sergiohinojosa there is this image that I maintain to be used in the mean time: https://github.com/dynatrace-ace/monaco-runner

**Reply by sergiohinojosa** [reply]
Awesome! thanks! 
FYI @henrikrexed
